source/direct_debit/index.html.md.erb

```diff
@@ -176,13 +176,13 @@ For example:
 
 The query parameters you can use to search are:
 
-- `reference`
-- `status`
-- `bank_statement_reference`
+- `reference` - exact match only
+- `status` - exact match only
```

Comment:
I know it was already like this but it’s actually `state`, not `status`.

Reply:
Thanks Alex - fixed in https://github.com/alphagov/pay-tech-docs/pull/320/commits/ec5793bfcb5ccd6af48b5f7977c3e98505ec0253.